native/android/.idea/jarRepositories.xml

```diff
@@ -61,5 +61,10 @@
       <option name="name" value="maven4" />
       <option name="url" value="file:$PROJECT_DIR$/../node_modules/react-native-screens/node_modules/react-native/android" />
     </remote-repository>
+    <remote-repository>
```

Comment:
Is this accidental/necessary?

Reply:
I think this is fine, not exactly sure about this file though